### PR TITLE
completed UI skeleton for the main page

### PR DESCRIPTION
**Pull request by bsoe003**
PR for Issue: #1
Please review and 'Squash and merge'.

In this PR:
* completed UI skeleton for the main page
* added autoprefixer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +102,9 @@ JobCompare application will have some specific guidelines for using git.
 ### Recommended Git Tools
 * [SourceTree](https://www.sourcetreeapp.com/) or [GitX](http://gitx.frim.nl/)
 
+## Useful Links
```
Comment (timothyk7) [inline README.md:105]: description of what this is please

File: app/components/Layout.js [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
+class Layout extends React.Component {
+  constructor() {
+    super();
+    this.state = { isMobile: false };
+  }
 
-      </section>
-    </div>
-  );
+  componentDidMount() {
```
Comment (timothyk7) [inline app/components/Layout.js]: maybe call it `componentPropMount` or `addComponentProp`?  `componentDidMount` makes it sound like a boolean method.

File: README.md (hunk 1 above)
Comment (timothyk7) [reply README.md:105]: caniuse

File: app/components/Layout.js [hunk 3]
```diff
@@ -1,19 +1,38 @@
...
-    </div>
-  );
+@Responsive
+class Layout extends React.Component {
+  constructor() {
+    super();
+    this.state = { isMobile: false };
+  }
+
+  componentDidMount() {
```
Comment (timothyk7) [inline app/components/Layout.js:15]: this is for adjusting screen for mobile right?  Maybe call it "compnentAdjustMobile"?

**Review by timothyk7 — DISMISSED** [review]
Fix

File: app/components/Layout.js (hunk 3 above)
Comment (bsoe003) [reply app/components/Layout.js:15]: `componentDidMount` is a built-in function in react.

**Review by timothyk7 — APPROVED** [review]

**Pull request merged**